functions/main.go

```diff
@@ -12,21 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:build netlify
-// +build netlify
-
 package main
```

Comment:
Unfortunately it would need to live outside of the `functions` directory, because sub-directories of `functions` are considered serverless functions for deploy. Could put it in `internal/cmd/serverless-local`?

Reply:
SGTM

Reply:
Done.